docs/GERMLINE_SV_README.md

```diff
@@ -0,0 +1,105 @@
+# Kids First Data Resource Center Germline Structural Variant Caller Workflow
+
+<p align="center">
+  <img src="https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png">
+</p>
+
+The Kids First Data Resource Center (KFDRC) Germline Structural Variant (SV)
+Caller Workflow is a common workflow language (CWL) implmentation to generate
+SV calls from an aligned reads BAM file. The workflow makes use of Manta and
```

Comment:
maybe generalize to aligned reads BAM/CRAM?

Reply:
Updated.